cmd/relayproxy/config/exporter.go

```diff
@@ -1,22 +1,27 @@
 package config
 
-import "fmt"
+import (
+	"fmt"
+
+	"github.com/xitongsys/parquet-go/parquet"
+)
 
 // ExporterConf contains all the field to configure an exporter
 type ExporterConf struct {
-	Kind             ExporterKind      `mapstructure:"kind" koanf:"kind"`
-	OutputDir        string            `mapstructure:"outputDir" koanf:"outputdir"`
-	Format           string            `mapstructure:"format" koanf:"format"`
-	Filename         string            `mapstructure:"filename" koanf:"filename"`
-	CsvTemplate      string            `mapstructure:"csvTemplate" koanf:"csvtemplate"`
-	Bucket           string            `mapstructure:"bucket" koanf:"bucket"`
-	Path             string            `mapstructure:"path" koanf:"path"`
-	EndpointURL      string            `mapstructure:"endpointUrl" koanf:"endpointurl"`
-	Secret           string            `mapstructure:"secret" koanf:"secret"`
-	Meta             map[string]string `mapstructure:"meta" koanf:"meta"`
-	LogFormat        string            `mapstructure:"logFormat" koanf:"logformat"`
-	FlushInterval    int64             `mapstructure:"flushInterval" koanf:"flushinterval"`
-	MaxEventInMemory int64             `mapstructure:"maxEventInMemory" koanf:"maxeventinmemory"`
+	Kind                    ExporterKind      `mapstructure:"kind" koanf:"kind"`
+	OutputDir               string            `mapstructure:"outputDir" koanf:"outputdir"`
+	Format                  string            `mapstructure:"format" koanf:"format"`
+	Filename                string            `mapstructure:"filename" koanf:"filename"`
+	CsvTemplate             string            `mapstructure:"csvTemplate" koanf:"csvtemplate"`
+	Bucket                  string            `mapstructure:"bucket" koanf:"bucket"`
+	Path                    string            `mapstructure:"path" koanf:"path"`
+	EndpointURL             string            `mapstructure:"endpointUrl" koanf:"endpointurl"`
+	Secret                  string            `mapstructure:"secret" koanf:"secret"`
+	Meta                    map[string]string `mapstructure:"meta" koanf:"meta"`
+	LogFormat               string            `mapstructure:"logFormat" koanf:"logformat"`
+	FlushInterval           int64             `mapstructure:"flushInterval" koanf:"flushinterval"`
+	MaxEventInMemory        int64             `mapstructure:"maxEventInMemory" koanf:"maxeventinmemory"`
+	ParquetCompressionCodec string            `mapstructure:"parquetcompressioncodec" koanf:"parquetCompressionCodec"`
```

Comment:
```suggestion
	ParquetCompressionCodec string            `mapstructure:"parquetCompressionCodec" koanf:"parquetcompressioncodec"`
```

Reply:
Everything should be lowercase in the koanf config to be able to be overridden in env variable.